Recommend not to use System.Drawing (#4594)
dotnet/dotnet-api-docs: includes/drawing.md

> [!WARNING]

> > We don't recommend that you use the `System.Drawing` namespace for new development as it is  not supported for use within a Windows or ASP.NET service and it is not cross-platform. Use  [ImageSharp](https://github.com/SixLabors/ImageSharp) or other libraries instead.

Reviewer: ```suggestion
> The `System.Drawing` namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service and it is not cross-platform. [ImageSharp](https://github.com/SixLabors/ImageSharp) and [SkiaSharp](https://github.com/mono/SkiaSharp) are recommended as alternatives.
```
